packages/Carousel/README.md

```diff
@@ -0,0 +1,80 @@
+# Carousel
+
+A carousel allows users to see and navigate the steps within a flow.
+
+<Example is="custom" />
+
+<ReferenceLinks is="custom" />
+
+## Installation
+
+To install `@igloo-ui/carousel` in your project, you will need to run the following command using [npm](https://www.npmjs.com/):
+
+```bash
+npm install @igloo-ui/carousel
+```
+
+If you prefer [Yarn](https://classic.yarnpkg.com/en/), use the following command instead:
+
+```bash
+yarn add @igloo-ui/carousel
+```
+
+## Usage
+
+Then to use the component in your code just import it!
+
+```jsx
+import Button from '@igloo-ui/button';
+import Carousel from '@igloo-ui/carousel';
+
+const SLIDE_NUM = 3;
+const [selected, setSelected] = React.useState(0);
+const [secondaryActionText, setSecondaryActionText] = React.useState('Cancel');
+const [primaryActionText, setPrimaryActionText] = React.useState('Next');
+
+const handlePageChange = (index) => {
+  if (index < SLIDE_NUM - 1) {
```

Comment:
This part of the doc. can be simplified by removing the `useState()` and placing the logic in the button.

Example: 
```
const handlePageChange = (index: number) => {
  setSelected(index);
};
```